emission/analysis/plotting/geojson/geojson_feature_converter.py

```diff
@@ -272,7 +274,8 @@ def get_geojson_for_timeline(user_id, tl):
                 logging.info("Skipping zero section trip %s with distance %s (should be zero)" %
                              (trip, trip_geojson.properties["distance"]))
             else:
-                geojson_list.append(trip_geojson)
+                logging.debug("adding %s to list" % json.loads(json_util.dumps(trip_geojson)))
+                geojson_list.append(trip_geojson) if not viz else geojson_list.append(trip_geojson.features)
```

Comment:
Can you explain why you need this? It seems like you can just iterate over `trip_geojson.features` instead of `trip_geojson` in `visualize.py`, and then you don't need to change the interface to `get_geojson_for_timeline`. Note that geojson is already a visualization format :)